spec.html

```diff
@@ -42927,7 +42927,9 @@ <h1>%AsyncFromSyncIteratorPrototype%.throw ( [ _value_ ] )</h1>
             1. Let _throw_ be GetMethod(_syncIterator_, *"throw"*).
             1. IfAbruptRejectPromise(_throw_, _promiseCapability_).
             1. If _throw_ is *undefined*, then
-              1. Perform ! Call(_promiseCapability_.[[Reject]], *undefined*, &laquo; _value_ &raquo;).
+              1. NOTE: If _syncIterator_ does not have a `throw` method, close it to give it a chance to clean up.
+              1. Set _valueRecord_ to IteratorClose(_syncIteratorRecord_, ThrowCompletion(_value_).
```

Comment:
Typo:
```suggestion
              1. Set _valueRecord_ to IteratorClose(_syncIteratorRecord_, ThrowCompletion(_value_)).
```

Also, since this is the line which introduces `valueRecord`, it should be

```suggestion
              1. Let _valueRecord_ be IteratorClose(_syncIteratorRecord_, ThrowCompletion(_value_)).
```

Reply:
Done. I'm surprised the formatter didn't catch the unbalanced parenthesis.

Reply:
The formatter doesn't really understand that level of detail, as yet; just the top-level HTML and algorithm steps.